test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -170,16 +170,16 @@ def test_qsub_with_option_a(self):
         # submit a job with future time and should start whenever time hits
         future_tm = time.strftime(
             "%Y%m%d%H%M", time.localtime(
-                present_tm + 60))
+                present_tm + 120))
         j1 = Job(TEST_USER, {ATTR_a: future_tm})
         try:
             jid_1 = self.server.submit(j1)
         except PbsSubmitError as e:
             self.validate_error(e)
         self.server.expect(JOB, {'job_state': 'W'}, id=jid_1)
         self.logger.info(
-            'waiting for 60 seconds to run the job as it is a future job...')
-        self.server.expect(JOB, {'job_state': 'R'}, id=jid_1, offset=60)
+            'waiting for 90 seconds to run the job as it is a future job...')
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid_1, offset=90)
```

Comment:
Since the job might have to wait up to 30s to run, consider adding an interval of 1 or 2s.